src/components/common/Alert.tsx

```diff
@@ -0,0 +1,88 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import starIcon from '@assets/star.svg';
+import Image from 'next/image';
+
+interface AlertProps {
+  message: String;
+  leftButtonTitle: String;
+  rightButtonTitle: String;
+  handleLeftClick?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+  handleRightClick?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+}
+
+function Alert(props: AlertProps) {
+  return (
```

Comment:
```suggestion
function Alert(props: AlertProps) {
  const { message, leftButtonTitle, rightButtonTitle, handleLeftClick, handleRightClick } = props
  return (
```

deconstructing을 활용하여 `props.` 이 반복되는 것을 줄이면 좋을거같아요!